core/common/src/main/java/alluxio/wire/ThriftUtils.java

```diff
@@ -83,6 +83,17 @@ public static FileInfo fromThrift(alluxio.thrift.FileInfo fileInfo) {
     return new FileInfo(fileInfo);
   }
 
+  /**
+   * Converts a thrift type to a wire type.
+   *
+   * @param inconsistentProperty the thrift representation of an inconsistent property
+   * @return wire representation of the inconsistent property
+   */
+  public static InconsistentProperty fromThrift(
```

Comment:
is this method necessary? Seems like the caller could save a few characters by just calling `new InconsistentProperty(inconsistentProperty)` themselves

Reply:
I am not sure why we have the conventions to use ThriftUtils to wrap all the fromThrift and toThrift methods in wire type.